python/src/aiconfig/default_parsers/openai.py

```diff
@@ -536,3 +581,41 @@ def is_prompt_template(prompt: Prompt):
     return isinstance(prompt.input, str) or (
         hasattr(prompt.input, "data") and isinstance(prompt.input.data, str)
     )
+
+def build_output_data(
+    message: Union[ChatCompletionMessage, None],
+) -> Union[OutputDataWithValue, str, None]:
+    output_data : Union[OutputDataWithValue, str, None] = None
+    if message is not None:
+        if message.get("content", None) is not None:
+            output_data = message.get("content")
+        elif message.get("tool_calls", None) is not None:
+            tool_calls = [ToolCallData(
```

Comment:
nit: this is fine for now, but we might need to update this in the future as `tool_calls` include things that aren't just function calls

Reply:
Good callout, I'll do another check where I'll check for the type being 'function'